net-irc/rhapsody/rhapsody-0.28b.ebuild

```diff
@@ -12,14 +12,13 @@ SRC_URI="https://downloads.sourceforge.net/${PN}/${PN}_${PV}.tgz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="amd64 ~riscv x86"
-IUSE=""
 
-DEPEND=">=sys-libs/ncurses-5.0:0="
 BDEPEND="virtual/pkgconfig"
+DEPEND=">=sys-libs/ncurses-5.0:0="
 
 PATCHES=(
-	"${FILESDIR}"/${P}-uclibc.patch
-	"${FILESDIR}"/${P}-tinfo.patch
+	"${FILESDIR}/${P}-uclibc.patch"
+	"${FILESDIR}/${P}-tinfo.patch"
```

Comment:
none of these are necessary, and BDEPEND actually comes last in the dependency blocks in skel.ebuild.